src/components/ProgressCard/index.tsx

```diff
@@ -39,6 +43,27 @@ export default function ProgressCard({ taskProgress }: Props) {
         }
     }, [taskProgress]);
 
+    const cardsToShow = sortedProgress?.data?.map((progress, idx) => {
+        if (idx === 0 && sortedProgress?.order === 1 && isDev) {
+            return (
+                <LatestProgressUpdateCard key={progress.id} data={progress} />
+            );
+        }
+
+        if (
+            sortedProgressLength &&
+            idx === sortedProgressLength - 1 &&
+            sortedProgress.order === 0 &&
+            isDev
+        ) {
+            return (
+                <LatestProgressUpdateCard key={progress.id} data={progress} />
+            );
+        }
+
+        return <ProgressDetails key={progress.id} data={progress} />;
+    });
```

Comment:
can we fix the indentation here?

Reply:
it is happening after I run yarn prettier-fix I think it's project's configuration